controllers/feeds/getFeedsV2.js

```diff
@@ -21,6 +21,7 @@ const {
 const {DomainPage, Locations, User} = require('../../databases/models');
 const RedisDomainHelper = require('../../services/redis/helper/RedisDomainHelper');
 const {ACTIVITY_THRESHOLD} = require('../../config/constant');
+const UsersFunction = require('../../databases/functions/users');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `UsersFunction` is a new addition and seems to be used correctly in the code. However, it's important to ensure that all methods from `UsersFunction` are properly exported and available for use.
```diff
+ const UsersFunction = require('../../databases/functions/users');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->